### PR TITLE
Simulator

### PR DESCRIPTION
**Pull request by ashwin2802**

### CODE REVIEW

**Comment by ashwin2802** [comment]
@Divyanshu23 @kpiyush25 have a good look.

File: simulator/launch/sim.launch [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version = "1.0" encoding="UTF-8"?>
+<launch>
+	<arg name="mav_name" default="firefly"/>
```
Comment (Kshitij1K) [inline simulator/launch/sim.launch:3]: fix indentations in this file

File: simulator/package.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  <maintainer email="dvyanshu@iitk.ac.in">Divyanshu Gangwar</maintainer>
+
+  <license>TODO</license>
+
+  <author email="dvyanshu@iitk.ac.in">Divyanshu Gangwar</author>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
```
Comment (Kshitij1K) [inline simulator/package.xml]: just put roscpp inside depend tag, instead of all these depends. these depends will be added automatically then.

**Review by Divyanshu23 — APPROVED** [review]

**Review by Divyanshu23 — DISMISSED** [review]

**Review by kpiyush25 — DISMISSED** [review]

**Review by Kshitij1K — APPROVED** [review]
Looks good to me

**Review by Divyanshu23 — APPROVED** [review]

**Pull request merged**